common/djangoapps/student/views.py

```diff
@@ -2133,7 +2133,7 @@ def do_email_change_request(user, new_email, activation_key=None):
 
 
 @ensure_csrf_cookie
-@transaction.commit_manually
+@transaction.non_atomic_requests
```

Comment:
Most of the commit_manually uses are replaced with atomic, but this one is non_atomic_requests.  Why is that?